File: ejercicio_24/ejercicio_23.java
package programacion_practica.ejercicio_23;
import java.util.Scanner;
public class ejercicio_23 {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);
        int[] list = new int[5];
        for(int i = 0; i < 5; i++){
            System.out.print("Ingresa el elemento "+(i+1)+" de la la lista: ");
            list[i] = entrada.nextInt();
        }
        int currentIndex = 0;  
        String input;
        do {
            System.out.print("Escribe 'next' para siguiente, 'prev' para anterior, o 'salir' para terminar: ");
            input = entrada.nextLine();
            if (input.equals("next")) {
                System.out.println("Elemento actual: " + list[currentIndex]);
                currentIndex = (currentIndex + 1) % list.length;  
            } else if (input.equals("prev")) {
                currentIndex = (currentIndex - 1 + list.length) % list.length; 
                System.out.println("Elemento actual: " + list[currentIndex]);
            }
        } while (!input.equals("salir"));
        entrada.close();
    }
    
}

File: ejercicio_24/ejercicio_23.py
list = []
for i in range(5):
    elemento = int(input(f"Ingresa el elemento {i+1} de la lista: "))
    list.append(elemento)
current_index = 0
input_usuario = ""
while input_usuario != "salir":
    input_usuario = input("Escribe 'next' para siguiente, 'prev' para anterior, o 'salir' para terminar: ")
    if input_usuario == "next":
        print(f"Elemento actual: {list[current_index]}")
        current_index = (current_index + 1) % len(list)
    elif input_usuario == "prev":
        current_index = (current_index - 1 + len(list)) % len(list)
        print(f"Elemento actual: {list[current_index]}")
